marklogic-client-api/src/main/java/com/marklogic/client/expression/PlanBuilderBase.java

```diff
@@ -151,7 +151,13 @@ public interface PlanBuilderBase {
      * @return  an AccessPlan object
      */
     PlanBuilder.AccessPlan fromLiterals(Map<String,Object>[] rows, XsStringVal qualifierName);
-
+    /**
+     * Provides a convinience equivalent to fromLexicons({uri: cts.uriReference()},qualifier,op.fragmentIdCol('fragmentId')).where(querydef).
```

Comment:
Typo - convinience -> convenience 

Also, for consistency with other javadocs in this class, I think "link" should be used - i.e.:

    equivalent to {@link PlanBuilder#fromLexicons(Map, String, PlanSystemColumn)}

And we'd want to reference that Java method as opposed to an SJS script, as the user may not be familiar at all with SJS.